index.js

```diff
@@ -3,29 +3,88 @@
  * then logs "Blast Off!"
  */
 
+ // function countdown(num) {
+ // 	for (var i = num; i >= 0; i--) {
+ // 		if (i === 0) { 
+ // 			console.log('Blast Off');
+ // 		} else {
+ // 			console.log(i);
+ // 		}
+ // 	}
+ // };
+
+function countdown(num) {
+	var i = num; 
+	while (i > 0) {
+		console.log (i--);
+	}
+	console.log('Blast Off');
+};
+
+countdown(10);
 
 /**
  * Q1: Write a function called `max` that has two Number parameters and returns
  * the larger of the two.
  */
 
+function max(x,y) {
+	return Math.max(x,y);
+};
+
+var larger = max(3,1);
+console.log(larger);
 
 /**
  * Q2: Write a function called `longer` that has two String parameters and returns
  * the String that is longer.
  */
 
+function longer(word1, word2) {
+	var wordLength = Math.max(word1.length, word2.length);
+	console.log(wordLength);
+
+	function whichOne(wordLength) {
+		if (wordLength == word1.legnth) {
+			return 'word1';
+		} else {
+			return 'word2';
+		}
+	}
```

Comment:
You've defined this inner helper function, but you never actually call it, so nothing will happen when we call the top level `longer` function. If you want to structure it this way, you'd need to call `whichOne` and return whatever that function returns:

```
function longer(word1, word2) {
    var wordLength = Math.max(word1.length, word2.length);
    function whichOne(len) {
        if (len === word1.length) {
            return word1;
        } else {
            return word2;
        }
    }
    return whichOne(wordLength);
}
```

However, this nested function is probably not needed as we can do the conditional directly in `longer`:

```
function longer(word1, word2) {
    var wordLength = Math.max(word1.length, word2.length);
    if (wordLength === word1.length) {
        return word1;
    } else {
        return word2;
    }
}
```